### ISSUE TITLE
Bug when sequence names are integer

### ISSUE THREAD

**Issue by simroux** [jiarong/VirSorter2#39]
Somewhat of an unexpected bug, took me a while to figure it out :-) 

If the sequence name is an integer, e.g.:
>1
GGGTTAATGTCGTTGGCTGGTATTTAATGAGAACGTGACAAGCGCCTCTCACGTTGCTAAGATTGGATTGC...

VirSorter2 performs all the steps without an error (and nothing appears in the logs, at least I couldn't find anything), however the final result files are empty (i.e. 0 virus predicted). The same sequence with a non-integer name (e.g. "Seq_1") get correctly predicted. I suspect python mis-interprets the sequence ID and then refuses to do some concatenation (because "you can't cat an int and a string" ... ... ...).

Nothing urgent to fix, but filing an issue (i) in case others have the same issue, and (ii) see if we can fix this in a future release (not sure how easy it will be to track down where the issue is exactly)

**Comment by jiarong** [comment]
Good catch. This one is not easy to notice.. The bug is caused by inconsistency in treating numbers as number or string. Fixed.

**Comment by hitig** [comment]
Finally, I found the solutions, thanks guys.
